e2e/wdio/core/tests/tabs.e2e-spec.ts

```diff
@@ -0,0 +1,156 @@
+        clickOnTab(collapsibleExample, 2);
+        expect(getAttributeByName(tabPanel, 'aria-expanded', 2)).toEqual('true');
+    });
+
+    describe('Tabs constructor testing', () => {
+        it('Check that tabs change according to chosen filter and compact modes', () => {
+            const myTittle = 'my custom title';
+            const myCount = 'my count';
+            const myContent = 'my content';
+            click(modeSelect)
+            click(filterMode)
+            click(compactCheckBox)
+            expect(getAttributeByName(threeElementsRow, 'ng-reflect-compact')).toEqual('true')
+            expect(getElementClass('playground .fd-tabs')).toContain('fd-tabs--filter')
+            setValue(titleField, myTittle); setValue(counterField, myCount); setValue(contentField, myContent);
+            expect(getText(titleAndCountSection)).toContain(myTittle)
+            expect(getText(titleAndCountSection)).toContain(myCount)
+            expect(getText(contentSection)).toEqual(myContent)
+        });
+        it('check that icon changes according to chosen', () => {
+            click(modeSelect)
+            click(iconOnlyMode)
+            click(compactCheckBox)
+            click(icon1)
+            click(acceleratedIcon)
+            expect(getElementClass(fdIcon)).toContain('sap-icon--accelerated')
+        });
+
+    });
+
+    it('should check examples visual regression', () => {
+        tabsPage.saveExampleBaselineScreenshot();
+        expect(tabsPage.compareWithBaseline()).toBeLessThan(5);
+    });
+
+    it('should check RTL and LTR orientation', () => {
+        tabsPage.checkRtlSwitch();
+    });
+
+    function checkTabsSelect(section: string): boolean {
+        let length;
+        if (section === collapsibleOverflowExample) {
+            length = getElementArrayLength(section + fdTab) - 3
+        }
+        if (section !== collapsibleOverflowExample) {
+            length = getElementArrayLength(section + fdTab)
+        }
+        for (let i = 0; i < length; i++) {
+            if (section === collapsibleExample && i === 0) {
+                click(section + fdTab, i)
+                if (getAttributeByName(section + fdTab, 'aria-selected', i) === 'true') {
+                    return false;
+                }
+            }
+            if (section === stackendContentExample && i === 0) {
+                expect(getAttributeByName(section + fdTab, 'aria-selected', i)).toEqual('true')
+            }
+            if (section !== collapsibleExample && section !== stackendContentExample && i !== 0) {
+                click(section + fdTab, i)
+                if (getAttributeByName(section + fdTab, 'aria-selected', i) !== 'true') {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    function clickOnTab(section: string, index: number = 0): void {
+        return (browserIsFirefox() ? click(section + 'fd-tabs__link', index): click(section + fdTab, index))
```

Comment:
add a space before `:`  and need to add dot before class name `.fd-tabs__link` -- you can also move that to po files and give it a name like `fdTabFF` so here will be this name instead of the string.